README.md

```diff
@@ -0,0 +1,140 @@
+# Codex surveys
+
+A simple constructor for the feedback widget.
+
+## How to use Surveys
+
+### Basics
+
+We have 2 global functions to use the widget:
+
+```javascript
+window.feedback = {
+  load: (configuration: NotionConfig) => Notion, // for loading Notion client data
+  createForm: (
+    configuration: {
+      form: FormConfig;
+      collapsedForm: CollapsedFormConfig;
+    },
+    sumbitEvent?: (data: Record<string, FormDataEntryValue>) => void
+  ) => void // create feedback form
+};
+```
+
+### Installation Guide
+
+1.  Create Notion [integration](https://www.notion.so/my-integrations)
+2.  Create Notion [database](https://www.notion.so/help/guides/creating-a-database)
+3.  Add connection to database
+    Right menu > Connections > Add connection
+4.  Installation:
+
+- For development:
+
+```bash
+    yarn && yarn dev
+```
```

Comment:
Remove this from the main readme since most users need only a usage guide, not development. We can collect developers' docs in a separate document.